pkg/cmd/opts/gocloud_integration_test.go

```diff
@@ -0,0 +1,38 @@
+package opts_test
+
+import (
+	"github.com/jenkins-x/jx/pkg/cmd/testhelpers"
+	"github.com/stretchr/testify/assert"
+	"path/filepath"
+	"testing"
+
+	"github.com/jenkins-x/jx/pkg/cmd/clients"
+	"github.com/jenkins-x/jx/pkg/cmd/opts"
+	"github.com/jenkins-x/jx/pkg/log"
+)
+
+func TestCommonOptions_EnsureGCloudBinaryInstalled(t *testing.T) {
+	o := opts.NewCommonOptionsWithFactory(clients.NewFactory())
+	o.NoBrew = true
+
+	origJXHome, testJXHome, err := testhelpers.CreateTestJxHomeDir()
+	if err != nil {
+		t.Errorf("Failed to create a test JX Home directory")
+	}
+
+	defer func() {
+		err := testhelpers.CleanupTestJxHomeDir(origJXHome, testJXHome)
+		if err != nil {
+			log.Logger().Warnf("Unable to remove temporary directory %s: %s", testJXHome, err)
+		}
+	}()
+
+	err = o.InstallGcloud()
+	if err != nil {
+		log.Logger().Warnf("Error in installing gcloud: %s", err)
+	}
+	
+	if !assert.FileExists(t, filepath.Join(testJXHome, "bin", "gcloud")) || err != nil {
```

Comment:
the assert is the opposite to what it should be and `assert.FileExists(t, filepath.Join(testJXHome, "bin", "gcloud")` should be sufficient.

Reply:
Doesn't `assert.FileExists` return true, so in this case if it returns false we throw the error ?